src/main/java/org/jfrog/hudson/JfrogServers.java

```diff
@@ -0,0 +1,77 @@
+package org.jfrog.hudson;
+
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Represents an instance of jenkins jfrog instance configuration page.
+ */
+public class JfrogServers {
+    private String platformUrl;
+    private String id;
+    private ArtifactoryServer artifactoryServer;
+    private CredentialsConfig deployerCredentialsConfig;
+    private CredentialsConfig resolverCredentialsConfig;
+
+    @DataBoundConstructor
+    public JfrogServers(String id, String artifactoryUrl, String platformUrl, CredentialsConfig deployerCredentialsConfig,
```

Comment:
platformUrl --> url
(let's remember to rename the getter and setter as well).

Also, url should come before artifactoryUrl in the method's arguments list.